search-sentiments-map/public/scripts/trends-script.js

```diff
@@ -57,21 +54,31 @@ function setTopTrends() {
 }
 
 /**
- * Shows the remaining 10 trending topics if the user clicks on 'See More' and
- * hides the last 10 if the user clicks on 'See Less.'
+ * Shows the trending topics over the 7th if the user clicks on 'See More' and
```

Comment:
I would replace the 7 to be more generalized.

Reply:
ok